common/xslt/dhq2html.xsl

```diff
@@ -249,13 +249,13 @@
         <!-- Select the chosen matching row -->
         <xsl:variable name="inner_tabs" select="tokenize($identifiedRow[1], '\t')" />
         <!-- Editing the url to get the relative url path. index 16 is the url-->
-        <xsl:variable name="path" select="substring-after($inner_tabs[16], 'org')" />
-        <!-- Now we pull from the tabs of the selected row. 5 is the name of the article -->
+        <xsl:variable name="path" select="substring-after($inner_tabs[17], 'org')" />
+        <!-- Now we pull from the tabs of the selected row. 6 is the name of the article -->
         <!-- Setting up the url and its <a> tag-->
-        <a href="{$path}"><xsl:value-of select="$inner_tabs[5]"/></a>,&#160;
-        <!-- selecting and inputting the year [2], author(s) [3], and university [4] -->
+        <a href="{$path}"><xsl:value-of select="$inner_tabs[6]"/></a>,
+        <!-- selecting and inputting the year [2], and author(s) and their affiliation [5] -->
         <xsl:value-of select="$inner_tabs[2]"/>,
```

Comment:
This is obviously fine. Two thoughts, though.

1. You would get nicer whitespace (for humans, browsers don’t care) if you used just `<xsl:value-of select="concat( $inner_tabs[2], ', ' $inner_tabs[5] )"/>`. (If we were using XSLT 3 it would be even nicer with just `<xsl:value-of select="$inner_tabs[2]||', '||$inner_tabs[5]"/>`.)
2. Is there a Style manual we are using? Now that the list of authors (probably with institutional affiliation) is a semicolon-separated list, I want the separator between the year and the author(s) to be stronger than a semicolon. (E.g., a period, a colon, a space-solidus-space, an em dash …)

Reply:
1. Ah nice! I didn't know that you could use concat like that.
2. Not sure if we have a Style manual, although it's a good point regarding the separator for the year. For what it's worth, in the issue page there is no year (because it is implied in the issue) and in the search results, there is a full line break between the year and the author-affiliation pairing. Maybe we could do that.

I'll go ahead and merge for now.